system/utils.py

```diff
@@ -315,7 +315,7 @@ def check_no_failures(hosts):
 
 
 async def ensure_pool_performs_write_read(
-    pool_handle, wallet_handle, trustee_did, nyms_count=1, timeout=30
+    pool_handle, wallet_handle, trustee_did, nyms_count=10, timeout=30
```

Comment:
Any particular reason for writing so much more NYMs?

Reply:
No, turned back to 1 everywhere.